src/components/molecules/MapPartygoerOverlay/MapPartygoerOverlay.tsx

```diff
@@ -43,6 +43,7 @@ export const MapPartygoerOverlay: React.FC<MapPartygoerOverlayProps> = ({
       top: `${avatarHeight * (currentRow - 1)}%`,
       left: `${avatarWidth * (currentCol - 1)}%`,
       justifyContent: "center",
+      zIndex: "12",
```

Comment:
Is the change added in `src/components/templates/PartyMap/components/Map/Map.scss` sufficient for this? If not, is there any reason we can't follow a similar pattern of setting the `z-index` layer within an scss class?